client/signup/storageUtils.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * Exernal dependencies
```

Comment:
Typo: Exernal -> External

Reply:
Thanks for catching that. I copied this line from utils.js, and I didn't notice the typo. I now fixed it in both files.